Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:bionic as builder
+
+RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
+    && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y bash vim.tiny wget sudo net-tools ca-certificates unzip apt-transport-https \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y python-minimal libmysqlclient-dev libxml2-dev libxslt-dev python-dev libffi-dev gcc libssl-dev gettext \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y python-pip python-setuptools nodejs node-gyp npm ruby nginx \
+    && pip install --upgrade pip \
+    && npm install -g less@2.7.1 \
+    && npm install -g yuglify@0.1.4 \
+    && gem install sass -v 3.4.22
+
+ARG DB_NAME=pol
```

Comment:
From my comment at https://github.com/taroved/pol/issues/26#issuecomment-533894971

> Also figured out that I should use env variables with an entrypoint.sh to replace Django settings on run-time instead of build time.

If we use ARGs, the DB connection will be "hardcoded" into the image, so whenever a CI will be set up, it leaks the password through the static image. We can keep the ARGs if we redefine them as ENVs below that take the ARGs as default value, so users can change them. However, introducing ENVs mean we need to do the settings.py sedding in the start.sh (runtime instead of build time).